config/routes.rb

```diff
@@ -7,8 +7,14 @@
     resources :url_rewrites
     resources :admin_users do
       resources :credentials, only: %i[new create destroy]
+      post :invite, on: :member, to: "tokens#create"
+      post :accept, on: :member, to: "tokens#update"
     end
 
+    # JWT tokens have dots(represents the 3 parts of data) in them, so we need to allow them in the URL
+    # can by pass if we use token as a query param
+    get "token/:token", to: "tokens#show", as: :token, token: /[^\/]+/
```

Comment:
TokenController can be inside admin/user, or inside session:

POST /admin/admin_users/token => creates a new URL and turbo-stream updates the response
PATCH /admin/sessions/token/:jwt => updates the user's session using the token to authenticate

Both can be inside the token controller, which doesn't need to have a namespace